todo_app/templates/index.html

```diff
@@ -10,19 +10,31 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
+      <ul class="list-group mb-2">
+        <form action="/" method="GET">
         <!-- List the todo items here -->
           <uL>
             {% for item in items %}
-              <li>{{ item.id }} - {{ item.status }} - {{ item.title }}</li>
+            <div class="mb-1">
+              <input type="button" value="Completed"  onclick="window.location.href='{{ url_for('completed', id=item.id) }}'">
+              <input type="button" value="Not Started" onclick="window.location.href='{{ url_for('not_started', id=item.id) }}'">
+              <input type="button" value="delete" onclick="window.location.href='{{ url_for('delete', id=item.id) }}'">
+              &emsp; {{ item.id }} - {{ item.status }} - {{ item.title }}
```

Comment:
If we do follow through on the suggested advice and switch to using Trello ids rather than a numerical count, then we might want to display a more user friendly value here; one approach would be to use [the in-built loop counter as a numerical counter](https://stackoverflow.com/questions/20233721/how-do-you-index-on-a-jinja-template):

```suggestion
              &emsp; {{ loop.index0 + 1 }} - {{ item.status }} - {{ item.title }}
```